packages/webapp/pages/plus/success.tsx

```diff
@@ -15,9 +15,21 @@ import {
 } from '@dailydotdev/shared/src/components/buttons/Button';
 import { webappUrl } from '@dailydotdev/shared/src/lib/constants';
 import { NextSeo } from 'next-seo';
+import { useBoot } from '@dailydotdev/shared/src/hooks';
+import { MarketingCtaVariant } from '@dailydotdev/shared/src/components/marketingCta/common';
 import { getPlusLayout } from '../../components/layouts/PlusLayout/PlusLayout';
 
 const PlusSuccessPage = (): ReactElement => {
+  const { getMarketingCta, clearMarketingCta } = useBoot();
+  const marketingCta = getMarketingCta(MarketingCtaVariant.Plus);
+
+  useEffect(() => {
+    if (marketingCta) {
+      const { campaignId } = marketingCta;
+      clearMarketingCta(campaignId);
+    }
+  }, [marketingCta, clearMarketingCta]);
```

Comment:
Kindly check whether these dependencies are memoized because if not, we might be firing the mutation that leads to sending analytics events.

Reply:
if we want to fire it only once we can use ref boolean, similar as in `Custom404Seo`